benchflow-dsl/src/main/scala/cloud/benchflow/dsl/definition/configuration/BenchFlowExperimentConfigurationYamlProtocol.scala

```diff
@@ -16,6 +16,10 @@ import scala.util.Try
  */
 object BenchFlowExperimentConfigurationYamlProtocol extends DefaultYamlProtocol {
 
+  // default value since deserializing from a test definition it might not be defined
+  // before use it will be overridden
+  val defaultUsers = 0
```

Comment:
change `defaultUsers` to `usersPlaceholder` and the value to `-1`

comment: "We use a placeholder since we need to have `users` defined in the experiment but it may not be defined in the test that is used to generate the template experiment. The template experiment is then changed according to the configuration of the exploration point. During the semantic check afterwards we check that the value has been changed."

open issue: should be checked in the semantic validation that it is no longer the template value